neko/mod.js

```diff
@@ -87,3 +96,4 @@ module.exports = {
     }
   }]
 }
+module.exports.settings.radio[0].options = options
```

Comment:
Would have like to do this in a more elegant way but the way the life cycle of the app works it's required to be here after everything else.